cmd/buildah/commit.go

```diff
@@ -20,6 +20,11 @@ var (
 			Usage: "don't compress layers",
 		},
 		cli.StringFlag{
+			Name:  "creds",
+			Value: "",
+			Usage: "Use `USERNAME[:PASSWORD]` for accessing the registry",
```

Comment:
good point.  This was a cut/paste job from skopeo and I didn't note that here.  All lower case now.